packages/openmrs/README.md

```diff
@@ -3,6 +3,8 @@
 Language Pack for building expressions and operations for working with the
 [OpenMRS API](https://wiki.openmrs.org/display/docs/API).
 
+[OpenMRS Rest API](https://rest.openmrs.org/?javascript#openmrs-rest-api)
```

Comment:
Is the link on line 4 still valid? Seems like this is the current API docs site and should replace the link on line 4. Thoughts?

Reply:
> Is the link on line 4 still valid? Seems like this is the current API docs site and should replace the link on line 4. Thoughts?

I think line 6 should replace line 4 as it directly takes the users to the API docs therefore reducing the number of clicks they will have to do to get to the actual API documentation